### DISCUSSION TITLE
Miss eventid in RDP

### DISCUSSION THREAD

**Discussion by ROBERTPASCAL** [yarox24/EvtxHussar#3] [General]
Hi,

Map RDP miss an EventID

Can you add 1158 eventID from source TerminalServices-RemoteConnectionManager ?

"Remote Desktop Services accepted a connection from IP adress XX.XX.XX.XX"

Thanks 

Best regards

**Comment by yarox24** [comment]
Hi Robert,

Yes, I would add this to next version.

Best regards
Jarek

**Comment by yarox24** [comment]
Commit: https://github.com/yarox24/EvtxHussar/commit/ea1eefdfd4991b7a69d3ca084467acd3af430860

**Reply by ROBERTPASCAL** [reply]
Great thanks !
